src/pages/index.js

```diff
@@ -94,8 +101,35 @@ console.log(currentUserInfo.getUserInfo());
 const profileEditModalPopup = new PopupWithForm(
   "#modal_profile-edit",
   (data) => {
-    currentUserInfo.setUserInfo(data.title, data.description);
-    profileEditModalPopup.close();
+    //currentUserInfo.setUserInfo(data.title, data.description);
+    //profileEditModalPopup.close();
+    //profileEditModalPopup.updating();
+    profileEditModalPopup.setSaveButtonText("Updating");
+
+    api
+      .editCurrentUser("/users/me", data)
+      .then(profileEditModalPopup.close())
+      .then(currentUserInfo.setUserInfo(data.title, data.description))
+      .catch((err) => console.error(err))
+      .finally(profileEditModalPopup.setSaveButtonText("Save"));
```

Comment:
Needs correction: `finally` method accepts a function (fix that everywhere)

Reply:
Done